app/views/reviews/_review.html.erb

```diff
@@ -1,36 +1,58 @@
-<div class="review review-<%= review.id %>">
-  <% unless current_page?(lifehack_path(review.lifehack)) %>
-    Lifehack: <%= link_to review.lifehack.title, lifehack_path(review.lifehack) %>
-  <% end %>
-  Rating: <%= review.rating %>
-  Comment: <%= review.comment %>
-  User: <%= link_to review.creator.full_name, user_path(review.creator) %>
-  Updated: <%= time_ago_in_words(review.updated_at)%> ago
 
-  <div class="score">
-    Score: <%= review.sum_score %>
+<div class="row">
+  <div class="review review-<%= review.id %>">
+    <table>
+      <thead>
+        <tr>
+          <th width="100">Votes</th>
+          <th width="200">Score </th>
+          <th width="200">Rating</th>
+          <th>Comment</th>
+          <th width="200">User</th>
+          <th width="400">Updated At</th>
+          <th width="250">Lifehack</th>
+        </tr>
+      </thead>
+      <tbody>
+      <tr class="rev_row">
+        <td>
+          <div class="row">
+            <% if user_signed_in? %>
```

Comment:
not sure if we still need this display logic, might be resolved in lawrence's PR though